### ISSUE TITLE
input validation

### ISSUE THREAD

**Issue by Abbbyccc** [fac25/week2-db-Sumithra-Joe-Paz-Abdullah#35]
can insert numerous anonymous data, not sure if its on purpose. 
![Screenshot 2022-09-29 at 17 04 41](https://user-images.githubusercontent.com/88027905/193081911-b67b01cb-5485-4ea3-886d-a07b03a3c371.png)

**Comment by Sumithra-Suresh** [comment]
Yes, it is on purpose. If the user doesn't want to add their name, we post the review as, 'anonymous'. 
Same for rating. It is defaulted to 0 to allow the user to skip the rating.

**Comment by ivanmauricio** [comment]
I also agree that there should be input validation. It's not best practice to allow a user to submit a form which they haven't interacted with at all. 

If you do want to allow this, I would recommend adding something like a pop-up where the user can confirm that they want to post anonymously with a rating of 0.
